cli/Valet/Brew.php

```diff
@@ -135,10 +145,14 @@ public function installOrFail(string $formula, array $options = [], array $taps
         }
 
         output('<info>['.$formula.'] is not installed, installing it now via Brew...</info> 🍻');
-        if ($formula !== 'php' && starts_with($formula, 'php') && preg_replace('/[^\d]/', '', $formula) < '73') {
+        if ($formula !== 'php' && starts_with($formula, 'php') && preg_replace('/[^\d]/', '', $formula) < '81') {
```

Comment:
Should we possibly move `81` here into a const so we can remember to check that regularly too?

Reply:
Since it's just displaying an informational message, ima merge it with the newly-added array constant, since both are actually quite tightly linked. And it's harmless if it shows the message but was wrong, since instead of 10 minutes it'll be 10 seconds, and nobody will be offput by that 🤷‍♂️

Reply:
I added a test as well